catalog/ui/catalog-ui-search/src/main/webapp/component/query-basic/query-basic.view.js

```diff
@@ -72,6 +72,16 @@ define([
     function translateFilterToBasicMap(filter){
         var propertyValueMap = {};
         var downConversion = false;
+
+        if (filter.filters && isAnyDate(filter)) {
```

Comment:
`filter.filters` is an array of filters within some (but not all filters). That expression evaluates to false if `filter.filters` is null and is true if it is anything else. If you look through the rest of the code, you'll see this kind of statement getting used a lot. I think this is an acceptable JS format, but would you know of a better way to write this?